docs/Select.mdx

```diff
@@ -89,3 +89,30 @@ import { Value } from 'react-powerplug'
     )}
   </Value>
 </Playground>
+
+## Error State
+
+<Playground>
+  <Value>
+    {({ value="bad", set }) => {
+      const error = value === "good" ? "" : "bad value"
```

Comment:
Should `"bad value" be "bad"? 

I think rather than "good" and "bad" semantics "Error" / "error" might be more appropriate since it maps to the prop thats being passed.

Reply:
How about "This is an error message set by the onChange handler"?

Reply:
😄  either way, the code is short and concise, it probably doesn't really matter much. My thinking about cleaning up bad is that other users would then see  "bad", "bad value", "error" and good and get confused documentation-wise. I think "error" is sufficient since its 1:1